src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1031,10 +1078,15 @@ def random_mac(self):
             return False
 
     @item_method
-    @accepts(Int('id'))
-    async def start(self, id):
+    @accepts(Int('id'), Dict('options', Bool('overcommit')))
```

Comment:
`Bool('overcommit', default=False)`

Reply:
This is master branch, different schema, am I dont want a default value, it should be None.

Reply:
Isnt that exactly the same compared to what we have now? What would be different?

Reply:
@william-gr in json schema it would be explicitly visible that default for `overcommit` is `null`, not `true` or `false`

Reply:
@themylogin how? at least in master branch we dont have that concept exposed in json schema?

```    def to_json_schema(self, parent=None):
        schema = {
            'type': ['boolean', 'null'] if not self.required else 'boolean',
        }
        if not parent:
            schema['title'] = self.verbose
            schema['_required_'] = self.required
        return schema```

Reply:
@william-gr probably we should do it in schema or other doc.